sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -161,7 +163,7 @@ def __init__(
                 retry_connect=self.connection_policy.ConnectionRetryConfiguration.connect,
                 retry_read=self.connection_policy.ConnectionRetryConfiguration.read,
                 retry_status=self.connection_policy.ConnectionRetryConfiguration.status,
-                retry_backoff_max=self.connection_policy.ConnectionRetryConfiguration.BACKOFF_MAX,
+                retry_backoff_max=self.connection_policy.ConnectionRetryConfiguration.backoff_max,  # type: ignore[attr-defined] # pylint:disable=line-too-long
```

Comment:
I don't know why mypy thinks a `urlib3.Retry` object doesn't have a `backoff_max` attribute.... when everything I look at tells me it does - so I've suppressed this for now...

Reply:
thank you!

Reply:
Turns out this was because the name changed between urllib3 v1 and v2. Given that we should support both, I've used the recommended replacement to `BACKOFF_MAX`, which fixes the mypy error.